src/main/resources/tnttime.mixins.json

```diff
@@ -6,7 +6,6 @@
   "compatibilityLevel": "JAVA_17",
   "mixins": [],
   "client": [
-    "CreeperEntityRendererMixin",
```

Comment:
CreeperEntityRenderer is nearly ready, I guess I will have to squash it to another branch for the time :)